src/com/company/Main.java

```diff
@@ -6,7 +6,7 @@
 
 public class Main {
 
-    public static boolean Related(String first, String second) {
+    public static boolean Realted(String first, String second) {
         first=first.toUpperCase();
         second=second.toUpperCase();
```

Comment:
Btw you can clarify if the question asks for case insensitive comparison or not, in this case I never mentioned if it is or is not